src/duckdb/spatial.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2023-2024 Retake, Inc.
+//
+// This file is part of ParadeDB - Postgres for Search and Analytics
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+use anyhow::{anyhow, Result};
+use std::collections::HashMap;
+
+#[derive(PartialEq)]
+pub enum SpatialOption {
+    Files,
+    SequentialLayerScan,
+    SpatialFilter,
+    OpenOptions,
+    Layer,
+    AllowedDrivers,
+    SiblingFiles,
+    SpatialFilterBox,
+    KeepWkb,
+}
```

Comment:
reference https://github.com/duckdb/duckdb_spatial/blob/main/docs/functions.md#st_read

Reply:
Thank you, could you add this as a comment in the code?